go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/kitex-contrib/registry-nacos
+
+go 1.16
+
+replace (
```

Comment:
why use replace?

when you use "go mod" in your local environment, you don't need to replace these package. just use the following code at the beginning of your subpackage's codes:

```
github.com/kitex-contrib/registry-nacos/registry
```